projects/igniteui-angular/src/lib/tree/tree.component.ts

```diff
@@ -327,6 +327,7 @@ export class IgxTreeComponent implements IgxTree, OnInit, AfterViewInit, OnDestr
         private selectionService: IgxTreeSelectionService,
         private treeService: IgxTreeService,
         private element: ElementRef<HTMLElement>,
+        private platform: PlatformUtil
```

Comment:
There are some failing tests due to the constructor changes. Those should be fixed.

Reply:
fixed the tests by adding the extra parameter.